pyaedt/edb_core/dotnet/database.py

```diff
@@ -65,17 +65,23 @@ def __getattr__(self, key):  # pragma: no cover
             except AttributeError:
                 raise AttributeError("Attribute not present")
 
-    def __init__(self, pdata):
-        self.dotnetobj = pdata.Geometry.PolygonData
-        self.edb_api = pdata
+    def __init__(self, pedb, api_object=None):
+        self._pedb = pedb
+        self.dotnetobj = pedb.edb_api.geometry.api_class.PolygonData
+        self.edb_api = api_object
 
     @property
     def api_class(self):  # pragma: no cover
-        """Return Ansys.Ansoft.Edb class object."""
+        """:class:`Ansys.Ansoft.Edb` class object."""
         return self.dotnetobj
 
+    @property
+    def arcs(self):
+        """Get list of Edb.Geometry.ArcData."""
```

Comment:
```suggestion
        ""List of Edb.Geometry.ArcData."""
```
Properties are to be described as nouns. Can we make the object a link?